cluster-autoscaler/FAQ.md

```diff
@@ -707,6 +707,38 @@ spec:
         args: ["sleep"]
 ```
 
+### How can I tune Cluster Autoscaler's performance for processing ProvisioningRequests?
+
+Cluster Autoscaler can be run in batch processing mode for CheckCapacity
+ProvisioningRequests. In this mode, Cluster Autoscaler processes multiple
+CheckCapacity ProvisioningRequests in a single iteration. This mode is useful for
+scenarios where a large number of CheckCapacity ProvisioningRequests
+need to be processed.
+
+However, enabling batch processing for CheckCapacity ProvisioningRequests can adversely
+affect the performance of processing other types of ProvisioningRequests and incoming pods
+since iterations where CheckCapacity ProvisioningRequests are processed will take longer
+and scale-ups for other types of ProvisioningRequests and incoming pods will not be processed
+during that time.
+
+#### Enabling Batch Processing
+
+1. **Cluster Autoscaler Version**: Ensure you are using Cluster Autoscaler version 1.31.1 or later.
```

Comment:
1.31.1 implies it's going be cherry-picked to 1.31 - I don't think this is likely to happen. We generally don't backport features, only bug-fixes. 

I think the first version to have this will be 1.32.0.

Reply:
Ack.